README.md

```diff
@@ -93,7 +93,7 @@ you type `gradle grade` in your terminal window.
 
 ## Testing GatorGrader
 
-GatorGrader uses [Pytest](https://docs.pytest.org/en/latest/) for testing.
+GatorGrader uses [Pytest](https://docs.pytest.org/en/latest/) for testing, alongside with [pytest-clarity](https://github.com/darrenburns/pytest-clarity) to increase readability of pytest output.
```

Comment:
This should read either `alongside pytest-clarity` or `along with pytest-clarity`, not `alongside with`. Additionally, perhaps `for increased` instead of `to increase` would be better as well

Reply:
I need to be more careful with my writing. Thanks.